modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -609,14 +606,6 @@ internal class DiscoverAdapter(
                     }
                 }
 
-                private fun updateHeader(
-                    recyclerView: RecyclerView,
-                    position: Int,
-                ) {
-                    val headerViewHolder = recyclerView.findViewHolderForAdapterPosition(0) as? HeaderViewHolder
-                    headerViewHolder?.updateVisibility(position != 1)
-                }
```

Comment:
This change also fixes an issue where the header image is removed when swiping back to it. 

Before the fix it looked like this:

<img  width="400" src="https://github.com/user-attachments/assets/d6a95f73-460b-4728-a607-0b95dcab6540" />